GoogleSignIn/Tests/Unit/GIDEMMSupportTest.m

```diff
@@ -57,6 +63,40 @@ @interface GIDEMMSupportTest : XCTestCase
 
 @implementation GIDEMMSupportTest
 
+- (void)testEMMSupportDelegate {
+  XCTestExpectation *emmErrorExpectation = [self expectationWithDescription:@"EMM AppAuth error"];
+
+
+  GIDEMMSupport *emmSupport = [[GIDEMMSupport alloc] init];
+  GIDFailingOIDAuthState *authState = [GIDFailingOIDAuthState testInstance];
+  GTMAuthSession *authSession = [[GTMAuthSession alloc] initWithAuthState:authState];
+  authSession.delegate = emmSupport;
+  GIDFakeFetcherService *fakeFetcherService = [[GIDFakeFetcherService alloc]
+                                                initWithAuthorizer:authSession];
+
+  NSURLRequest *request = [NSURLRequest requestWithURL:[NSURL URLWithString:@""]];
+  GTMSessionFetcher *fakeFetcher = [fakeFetcherService fetcherWithRequest:request];
+
+  GIDGoogleUser *user = [[GIDGoogleUser alloc] init];
+  GIDTestWorker *testWorker = [[GIDTestWorker alloc] initWithGoogleUser:user
+                                                                fetcher:fakeFetcher];
```

Comment:
Not sure `GIDTestWorker` is necessary.

`GIDGoogleUser` is initialized using `init`, which isn't overridden in `GIDGoogleUser`, so it doesn't connect the instance with `GIDEMMSupport`. And `GIDTestWorker` does not do anything to the input `user` either.

Why not just call `[fakeFetcher beginFetchWithCompletionHandler:]` directly?

Reply:
Sorry this test was unclear. I've made some changes to hopefully make things more clear and simultaneously closer to the intended use-case (by our clients, who will use `-[GIDGoogleUser fetcherAuthorizer]` to pass to their own `GTMSessionFetcher` instances.

The goal for `GIDTestWorker` was to simulate some toy class that a client may have that does some work. In this case, all it does is fail, but that is by design so that we can test (and document) the path by which an EMM error case occurs and is handled.

Please take a look and comment or resolve as appropriate.

Reply:
I've removed `GIDTestWorker`.